src/bootstrap/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Changes since the last major version]
 
```

Comment:
```suggestion

## [Version 3] - 2022-10-20

```

Reply:
@jyn514 done, but somehow still the build is not going thru.

Reply:
Did you see the changes I suggested in https://github.com/rust-lang/rust/pull/103286#pullrequestreview-1149659205 ?

Reply:
@jyn514 seems some issue with extraction of file to bootstrap-sysroot. 

```
warning: there have been changes to x.py since you last updated.
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, update `config.toml` to use `changelog-seen = 3` instead
downloading https://static.rust-lang.org/dist/2022-09-20/rustfmt-nightly-aarch64-apple-darwin.tar.xz
############################################################################################################################################################################################################### 100.0%
extracting /Users/chetankokil/git/rust/build/cache/2022-09-20/rustfmt-nightly-aarch64-apple-darwin.tar.xz to /Users/chetankokil/git/rust/build/aarch64-apple-darwin/bootstrap-sysroot


failed to execute command: "/Users/chetankokil/git/rust/build/aarch64-apple-darwin/bootstrap-sysroot/bin/rustc" "--print" "sysroot"
error: No such file or directory (os error 2)
```.

This is not allowing me to build or push

Reply:
```
ls build/aarch64-apple-darwin build/aarch64-apple-darwin/bootstrap-sysroot/bin
ls: build/aarch64-apple-darwin/bootstrap-sysroot/bin: No such file or directory
build/aarch64-apple-darwin:
bootstrap-sysroot
```

Above is the output of `ls build/aarch64-apple-darwin build/aarch64-apple-darwin/bootstrap-sysroot/bin`